### PR TITLE
Refactor AccountProfile component

### PR DESCRIPTION
**Pull request by erditkurteshiSQA**
This Pull Request suggests some changes in the `AccountProfile` file to address the issue https://github.com/near/near-discovery/issues/853.

- Added `returnProfileForUser` function.
- Updated `TimeAgo` widget.
- Added prop `profile: returnProfileForUser(profile)` in AccountProfileOverlay widget.

### CODE REVIEW

File: src/AccountProfile.jsx [hunk 1]
```diff
@@ -1,9 +1,36 @@
 const accountId = props.accountId || context.accountId;
-const profile = props.profile || Social.get(`${accountId}/profile/**`, "final");
```
Comment (calebjacob) [inline src/AccountProfile.jsx:2]: If we're going to remove the fallback `Social.get(`${accountId}/profile/**`, "final")` logic, we'll need to make sure every use of `<AccountProfile>` is passing in the `profile` prop. I believe there are quite a few areas in our app that don't pass in the `profile` prop to the component and rely on this fallback logic to fetch it based on `accountId`.

**Review by calebjacob — CHANGES_REQUESTED** [review]
Overall this looks cleaner - nice job. See my comment about removing ```Social.get(`${accountId}/profile/**`, "final")```

File: src/AccountProfile.jsx (hunk 1 above)
Comment (shelegdmitriy) [reply src/AccountProfile.jsx:2]: I think it's worth to use `Social.get(`${accountId}/profile/**`, "final")` here

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]
Overall LGMT

File: src/AccountProfile.jsx (hunk 1 above)
Comment (erditkurteshiSQA) [reply src/AccountProfile.jsx:2]: Thank you for the reviews, I reverted the changes for `Social.get`

**Review by calebjacob — APPROVED** [review]

**Review by shelegdmitriy — APPROVED** [review]

**Pull request merged**